lib/screenplay/alerts/alert.ex

```diff
@@ -111,30 +112,31 @@ defmodule Screenplay.Alerts.Alert do
   # High priority (deliver to T-Alert subscribers immediately) -> 10
   def interpret_severity(severity) do
```

Comment:
I think you're right that severity 1 is different, but in reality we never show delay alerts with that low a severity level.
We [filter out](https://github.com/mbta/screenplay/blob/master/lib/screenplay/alerts/alert.ex#L128) delays with severity < 4.

This module is interesting because most of the code was simply copy-pasted from Screens, which we historically haven't been as thorough about testing... :sweat_smile: I'll add some!

Reply:
Tests added! I moved alert_test.exs from test/screenplay/ to test/screenplay/alerts as well, to have it properly match the path of the file it tests (lib/screenplay/alerts/alert.ex).